catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -753,6 +762,42 @@ public void setShouldConvertToBBox(boolean shouldConvertToBBox) {
     this.shouldConvertToBBox = shouldConvertToBBox;
   }
 
+  /**
+   * Get the number of verticies an approximation polygon will have when converting a multi
+   * point-radius search to a multi-polygon search.
+   */
+  public int getNumMultiPointRadiusVertices() {
+    return numMultiPointRadiusVertices;
+  }
+
+  /**
+   * Sets the number of verticies to use when approximating a polygon to fit to a multi point-radius
+   * search.
+   */
+  public void setNumMultiPointRadiusVertices(int numMultiPointRadiusVertices) {
+    this.numMultiPointRadiusVertices = numMultiPointRadiusVertices;
+  }
+
+  /**
+   * Get the number of verticies an approximation polygon will have when converting a multi
+   * point-radius search to a multi-polygon search.
+   */
+  public int getDistanceTolerance() {
+    return distanceTolerance;
+  }
+
+  /**
+   * Sets the number of verticies to use when approximating a polygon to fit to a multi point-radius
+   * search.
+   */
+  public void setDistanceTolerance(int distanceTolerance) {
+    if (distanceTolerance < MIN_DISTANCE_TOLERANCE) {
```

Comment:
✏️ `this.distanceTolerance = distanceTolerance < MIN_DISTANCE_TOLERANCE ? MIN_DISTANCE_TOLERANCE : distanceTolerance`